docs/cluster-resolver.md

```diff
@@ -74,6 +74,60 @@ spec:
       value: namespace-containing-pipeline
 ```
 
+## `ResolutionRequest` Status
+`ResolutionRequest.Status.Source` field captures the source where the remote resource came from. It includes the 3 subfields: `url`, `digest` and `entrypoint`.
+- `url`: [Kubernetes CRD namespace-scoped resource URI](https://kubernetes.io/docs/reference/using-api/api-concepts/#resource-uris) appended with UID i.e. `/apis/GROUP/VERSION/namespaces/NAMESPACE/RESOURCETYPE/NAME@UID`
+- `digest`: sha256 checksum of the content in the in-cluster resource's spec field. The reason why it's the checksum of the spec content rather than the whole object is because the metadata of in-cluster resources might be modified i.e. annotations. Therefore, the checksum of the spec content should be sufficient for source verifiers to verify if things have been changed maliciously even though the metadata is modified with good intentions.
+- `entrypoint`: ***empty*** because the path information is already available in the url field.
```

Comment:
Heyy @wlynch ,

I've changed to store the checksum of the spec of in-cluster task/pipeline as we discussed before, and also documented this here. Please take a look and please let me know if the documentation needs to be rephrased here.
Thanks!!